lib/bibliothecary/multi_parsers/cyclonedx.rb

```diff
@@ -85,7 +85,11 @@ def self.mapping
       end
 
       def parse_cyclonedx_json(file_contents, options: {})
-        manifest = JSON.parse(file_contents)
+        manifest = nil
+
+        manifest = try_cache(options, :cyclonedx_json) do
```

Comment:
is `options` shared among multiple files? Should the cache key have the path in it?